apps/i18n/common/en_us.json

```diff
@@ -2018,6 +2018,7 @@
   "privacyPolicy": "Privacy Policy",
   "privacyPracticesForChildren": "See our privacy practices for children",
   "projectWarning": "Note: You are on a level that is part of a longer project. Changes made on this level will also appear in other levels in the project.",
+  "promptIconAndLightbulbInstructionHints": "A friendly character with lightbulb image - click to display instruction hints prompt (Do you want a hint? yes or no) if hints are available",
```

Comment:
@samantha-code  - lmk what you think of this aria-label. Thanks!

Reply:
Perfect Label!